missingNumbers.js

```diff
@@ -1,25 +1,23 @@
 function findMissingNumbers(inputArray) {
-
     var missingNumbers = [];
-    if (typeof inputArray === 'string') {
-        return 'invalid input'
+    if (!Array.isArray(inputArray)) {
+        // used return for testing since thrown error couldnt be seen by the test
+        return 'Invalid Input'
     }
     if (inputArray.length <= 1) {
         return 'array length too small'
```

Comment:
Throw the error and modify the tests as discussed [here ](https://github.com/SeR4h/andelaweek3challenges1/pull/2#discussion_r359166870)